development/verify-locale-strings.js

```diff
@@ -187,6 +187,8 @@ async function verifyEnglishLocale() {
       'shared/**/*.js',
       'shared/**/*.ts',
       'shared/**/*.tsx',
+      'app/scripts/lib/**/*.js',
```

Comment:
Since we now have translations from the background scripts (which I added in my snap-keyring.ts file), we need to also check these paths to verify locale use.